### ISSUE TITLE
Fail CI if conventions are not followed

### ISSUE THREAD

**Issue by pepibumur** [tuist/tuist#781]
## Context 🕵️‍♀️
As mentioned by @andreacipriani [in this comment](https://github.com/tuist/tuist/pull/770#discussion_r355393796), PRs that don't follow the naming conventions are not reported as failing.

## What 🌱
- Double-check if the linting workflows are doing their job.

**Comment by lakpa** [comment]
**swiftformat** is not configured to run in our listing workflow currently, if this is something we are looking to integrate @pepibumur, I could help if needed

**Comment by pepibumur** [comment]
> swiftformat is not configured to run in our listing workflow currently unless i am missing something, if this is something we are looking to integrate @pepibumur, I could help if needed

Yes @lakpa. We should run that too. We could run it in such a way that if it fails, we tell developers to run `bundle exec rake style_correct`. What do you think?

**Comment by lakpa** [comment]
That could work, and as you have mentioned in the PR that having **github action** to run swift format will be ideal going forward. However for the time being, i am thinking of getting `style_correct` integrated as you have seen in my PR, it appears as if it needs `swiftlint` and `swiftformat` to be present but since our repo doesn't have it, could that be a reason the github action is failing?

**Comment by pepibumur** [comment]
`style_correct` is intended to be used locally, not on CI. On the `checks` GitHub workflow we should have 2 steps, one that runs SwiftLint, and another one that runs Swiftformat. Before running those, we need to make sure that SwiftLint and Swiftformat are installed, and that's something you can do by running `brew install xxx`.
